src/main/java/com/github/alex1304/jdash/util/Utils.java

```diff
@@ -132,6 +138,36 @@ public static String b64Decode(String str) {
 	public static String b64Encode(String str) {
 		return Base64.getUrlEncoder().encodeToString(str.getBytes());
 	}
+
+	public static String sha1Encrypt(String str) {
+		try {
+			MessageDigest md = MessageDigest.getInstance("SHA-1");
+			md.update(str.getBytes(StandardCharsets.UTF_8));
+			byte[] digest = md.digest();
+			StringBuilder sb = new StringBuilder();
+			for(byte b : digest) {
+				sb.append(String.format("%02x", 0xff & b));
+			}
+			return sb.toString();
+		} catch (NoSuchAlgorithmException e) {
+			return str;
```

Comment:
This should never happen so you can `throw new AssertionError()`. Not a good practice to swallow errors like that.

Reply:
Changed to throwing AssertionError 60055b5